packages/cli/src/bin.ts

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env node
 
 import './tools/gracefulifyFs';
+import semver from 'semver';
+import chalk from 'chalk';
 
-import {run} from './';
-
-run();
+if (semver.satisfies(process.version, '>=14')) {
```

Comment:
Maybe worth taking it from `versionRanges.ts` file?  🤔 
https://github.com/react-native-community/cli/blob/b06993e72dccc2658551b76a90b9f063e9cb8731/packages/cli-doctor/src/tools/versionRanges.ts#L3

Reply:
even better

Reply:
fixed ✅